modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -164,6 +166,25 @@ const Quirks = (editor: Editor): Quirks => {
     }
   };
 
+  const manageEnterInFigure = () => {
```

Comment:
There is a `keyboard/EnterKey` module that we call this function in also I think we could make this a generic thing not specific to `figure` elements since any non-editable element that has a editable host child element could have the same behaviour then we get this for free on other components.

Reply:
changed